qemu/tests/throttle_utils.py

```diff
@@ -0,0 +1,111 @@
+import logging
```

Comment:
put throttle_utils.py into provider dir? provider holds the utils or objects for common use

Reply:
will update